File: java/Baju.java
class Baju extends Aksesoris {
    private String kategori;
    private String size;
    private String merk;

    public Baju() {
    }

    public Baju(String inputKategori, String inputSize, String inputMerk) {
        this.kategori = inputKategori;
        this.size = inputSize;
        this.merk = inputMerk;
    }

    public void setKategori(String inputKategori) {
        this.kategori = inputKategori;
    }

    public String getKategori() {
        return this.kategori;
    }

    public void setSize(String inputSize) {
        this.size = inputSize;
    }

    public String getSize() {
        return this.size;
    }

    public void setMerk(String inputMerk) {
        this.merk = inputMerk;
    }

    public String getMerk() {
        return this.merk;
    }
}

File: php/Baju.php
<?php
include "Aksesoris.php";

class Baju extends Aksesoris
{
    private string $kategori;
    private string $size;
    private string $merk;

    function __construct(string $inputID, string $inputNama, int $inputHarga, int $inputStok, string $inputGambar, string $inputJenis, string $inputBahan, string $inputWarna, string $inputKategori, string $inputSize, string $inputMerk)
    {
        parent::__construct($inputID, $inputNama, $inputHarga, $inputStok, $inputGambar, $inputJenis, $inputBahan, $inputWarna);
        $this->kategori = $inputKategori;
        $this->size = $inputSize;
        $this->merk = $inputMerk;
    }

    public function setKategori(string $inputKategori): void
    {
        $this->kategori = $inputKategori;
    }

    public function getKategori(): string
    {
        return $this->kategori;
    }

    public function setSize(string $inputSize): void
    {
        $this->size = $inputSize;
    }

    public function getSize(): string
    {
        return $this->size;
    }

    public function setMerk(string $inputMerk): void
    {
        $this->merk = $inputMerk;
    }

    public function getMerk(): string
    {
        return $this->merk;
    }

    function __destruct() {}
}
